Description: remove from CFLAGS what gets passed by Debian build system
Forwarded: no
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-07-03

--- a/src/Makefile
+++ b/src/Makefile
@@ -9,7 +9,7 @@
 
 LIBPATH = -L.
 LDFLAGS +=  -lz `sh osflags $(TARGETOS) link` $(LIBPATH)
-CFLAGS += -c -g -Wall -D$(OS) -pedantic `sh osflags $(TARGETOS) cflags`
+CFLAGS += -c -D$(OS) -pedantic `sh osflags $(TARGETOS) cflags` $(CPPFLAGS)
 
 all: stateos $(CLIENT) $(SERVER)
 
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -7,7 +7,7 @@
 
 CHECK_PATH = /usr/local
 LDFLAGS = -L$(CHECK_PATH)/lib -lcheck `../src/osflags link`
-CFLAGS = -g -Wall -D$(OS) -I../src -I$(CHECK_PATH)/include -pedantic `../src/osflags cflags`
+CFLAGS += -D$(OS) -I../src -I$(CHECK_PATH)/include -pedantic `../src/osflags cflags` $(CPPFLAGS)
 
 all: $(TEST)
 	@LD_LIBRARY_PATH=${CHECK_PATH}/lib ./$(TEST)
